Makefile

```diff
@@ -35,6 +35,9 @@ IMAGE_TAG_BASE ?= ghcr.io/qbarrand/oot-operator
 # You can use it as an arg. (E.g make bundle-build BUNDLE_IMG=<some-registry>/<project-name-bundle>:<tag>)
 BUNDLE_IMG ?= $(IMAGE_TAG_BASE)-bundle:v$(VERSION)
 
+# GATHER_IMG define the must-gather image used to get all the information for debugging purpose.
+GATHER_IMG ?= quay.io/rh_ee_erusso/kmmo-must-gather
```

Comment:
Let's instead try to have it built by the GitHub CI and store it there.
Could you have a look at `.github/workflows/container-image.yml` and replicate what we do for other images?